roles/ucp/tasks/main.yml

```diff
@@ -32,6 +32,10 @@
     - "{{ ucp_instance_id_file }}"
   when: node_name != ucp_bootstrap_node_name
 
+- name: copy the ucp license to the remote machine
+  copy: src={{ ucp_license_file }} dest={{ ucp_license_remote }}
```

Comment:
> UCP needs a license in all cases, a trial license at minimal and so I had made it mandatory

ah. I was under the impression that it can work without a license. Just that user will only be able to bring up just one node with it. Isn't that the case anymore? If not, then you are right the changes you have are correct.